Engines/Wine/QuickScript/GoG Script/script.js

```diff
@@ -31,54 +33,65 @@ class GogScript extends QuickScript {
     /**
      * Presents a Gog.com login window to the user, login to its account and return a token that can be used later.
      * Stores the tocken in a parameter
+     *
      * @param {SetupWizard} setupWizard The setupWizard to use
      * @returns {GogScript} This
      */
     loginToGog(setupWizard) {
-        const browserWindow = setupWizard.createBrowser(tr("Please login to your GoG.com account so that we can download the game for you:"));
-        browserWindow.goToUrl("https://auth.gog.com/auth?client_id=46899977096215655&redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient&response_type=code&layout=client2");
+        const browserWindow = setupWizard.createBrowser(
+            tr("Please login to your GoG.com account so that we can download the game for you:")
+        );
+        browserWindow.goToUrl(
+            "https://auth.gog.com/auth?" +
+                "client_id=46899977096215655&" +
+                "redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient&" +
+                "response_type=code&" +
+                "layout=client2"
+        );
         browserWindow.waitForUrl("https://embed.gog.com/*");
 
         const currentUrl = browserWindow.getCurrentUrl();
         const code = currentUrl.split("code=")[1].split("&")[0];
 
-        const tokenUrl = "https://auth.gog.com/token?client_id=46899977096215655&client_secret=9d85c43b1482497dbbce61f6e4aa173a433796eeae2ca8c5f6129f2dc4de46d9&grant_type=authorization_code&code=" + code + "&redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient";
+        const tokenUrl =
+            "https://auth.gog.com/token?" +
+            "client_id=46899977096215655&" +
+            "client_secret=9d85c43b1482497dbbce61f6e4aa173a433796eeae2ca8c5f6129f2dc4de46d9&" +
+            "grant_type=authorization_code&" +
+            `code=${code}&` +
+            "redirect_uri=https%3A%2F%2Fembed.gog.com%2Fon_login_success%3Forigin%3Dclient";
         this._token = new Downloader().url(tokenUrl).json();
+
         return this;
     }
 
     _downloadSetupFile(setupWizard, setupFileName, tmpDirectory) {
-        const url = "https://www.gog.com/downloads/" + setupFileName;
+        const url = `https://www.gog.com/downloads/${setupFileName}`;
 
         // We set a user agent so that GoG sends the filename of the executable
         return new Downloader()
             .url(url)
             .wizard(setupWizard)
             .to(tmpDirectory)
             .headers({
-                "Authorization": "Bearer " + this._token["access_token"],
+                Authorization: "Bearer " + this._token["access_token"],
```

Comment:
Why not "" around Authorization anymore?

Reply:
Basically because auto-formatting removed it. It is also unnecessary in this case, should I add it again?

Reply:
I think yes for better understanding (because the next line uses "").

Reply:
Done